meteor/client/ui/Shelf/AdLibPanel.tsx

```diff
@@ -487,6 +487,7 @@ export function fetchAndFilter(props: IFetchAndFilterProps): AdLibFetchAndFilter
 								}
 							)
 								.fetch()
+								.filter((action) => !(action.publicData && ((action.publicData as any).template as boolean)))
```

Comment:
`publicData` is a blueprints-defined blob, I don't think Core should impose any rules on what various properties can be set there. I'm quite sure someone might want to set `publicData.template` to be the name of a graphics template that the AdLib is using. I don't think they would expect the AdLib to disappear from the Shelf because of that.

```suggestion
```

Reply:
Updated as suggested